tests/unit/linalg/test_hypre_vector.cpp

```diff
@@ -38,11 +32,30 @@ TEST_CASE("HypreParVector I/O", "[Parallel], [HypreParVector]")
    else
    {
       col.resize(world_size+1);
-      for (int i=0; i<world_size; ++i)
+      for (int i=1; i<=world_size; ++i)
```

Comment:
Good catch on the `<=`. Shouldn't it still start at 0 though?

Reply:
I believe it shouldn't make a difference since that first element will always be zero anyways, but for clarity I agree that the loop should start at zero.